src/main/java/com/epam/emotionalhelp/repository/QuizResultRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.emotionalhelp.repository;
+
+import com.epam.emotionalhelp.model.QuizResult;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.Collection;
+
+public interface QuizResultRepository extends JpaRepository<QuizResult, Long> {
+    Collection<QuizResult> findAllById(Long resultId);
+
+    @Query(value = "SELECT MAX(id) + 1 FROM QuizResult")
+    Long getNextRowId();
```

Comment:
We have agreed with @GeloPakDev to create a new `quiz_attempt` table that will store actual quiz attempt - please consider it in your implementation if needed.

See [Database model](https://github.com/emotional-help-competition/backend/wiki/Database-model) for more details.